The <asm/param.h> header is Linux-only and breaks GNU/Hurd builds.

--- util-linux-2.20.1/term-utils/write.c	2012-03-07 23:06:14.000000000 +0100
+++ util-linux-2.20.1/term-utils/write.c	2012-03-07 23:06:19.000000000 +0100
@@ -57,7 +57,9 @@
 #include <sys/param.h>
 #include <sys/stat.h>
 #include <paths.h>
-#include <asm/param.h>
+#ifdef __linux__
+# include <asm/param.h>
+#endif
 #include <getopt.h>
 #include "c.h"
 #include "carefulputc.h"

